src/plugins/mer/mersettings.cpp

```diff
@@ -115,6 +116,60 @@ void MerSettings::setRpmValidationByDefault(bool byDefault)
     emit s_instance->rpmValidationByDefaultChanged(s_instance->m_rpmValidationByDefault);
 }
 
+QHash<QString, MerEmulatorDeviceModel> MerSettings::customEmulatorDeviceModels()
+{
+    Q_ASSERT(s_instance);
+
+    return s_instance->m_customEmulatorDeviceModels;
+}
+
+bool MerSettings::isCustomEmulatorDeviceModel(const QString &name)
```

Comment:
Similar as with the now-removed setCustomEmulatorDeviceModel, I would prefer not introducing this function.

Reply:
This functions moved to MerEmulatorDevice:
`bool isCustomModel(const QString &name) const;`
`bool createCustomModel(const QString &name);`
 `bool deleteCustomModel(const QString &name);`